lib/js/components/SelectList/SelectListItem/SelectListItem.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<div v-ripple class="selectListItem" :class="{ '-selected': isSelected, [`-${size}`]: true }">
+		<ds-icon
+			v-if="iconLeft"
+			class="selectListItem__icon"
+			:icon="iconLeft"
+			:size="ICON_SIZES.X_SMALL"
+			:spinning="state === SELECT_LIST_ITEM_STATES.LOADING"
+		/>
+		<span class="selectListItem__text">{{ label }}</span>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/spacings';
+
+.selectListItem {
+	@include label-l-default-regular;
+
+	background-color: $color-neutral-background-ghost;
+	color: $color-neutral-text-heavy;
+	display: flex;
+	padding: $space-xs;
+	pointer-events: none;
+
+	&.-selected {
+		background-color: $color-neutral-background;
+	}
+
+	&.-medium {
+		@include label-xl-default-regular;
+	}
+
+	&:not(.-selected) {
+		cursor: pointer;
+		pointer-events: auto;
+
+		&:focus {
+			background-color: $color-neutral-background-ghost-focused;
+		}
+
+		&:hover {
+			background-color: $color-neutral-background-ghost-hovered;
+		}
+	}
+
+	&::v-deep .ripple {
+		background-color: $color-neutral-ripple !important;
+	}
+
+	&__icon {
+		color: $color-neutral-icon-strong;
+		margin-right: $space-xxxs;
+	}
+
+	&__text {
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
+}
+</style>
+
+<script lang="ts">
+import Ripple from 'vue-ripple-directive';
+
+import { SELECT_LIST_ITEM_STATES, SELECT_LIST_SIZES } from '../SelectList.consts';
+import DsIcon, { ICON_SIZES, ICONS } from '../../Icons/Icon';
+
+export default {
+	name: 'SelectListItem',
+	components: {
+		DsIcon,
+	},
+	directives: {
+		ripple: Ripple,
+	},
+	props: {
```

Comment:
I see many props in Miro, that aren't implemented here. Was it planned or just missed?

Reply:
In Miro and Figma v2 scope is incorrect. I implemented only the things we need for v2. Related thread: https://bethinkteam.slack.com/archives/C012R8RM3A4/p1677171070976499